COLLABORATOR_GUIDE.md

```diff
@@ -87,12 +86,54 @@ where a pull request:
 
 - has a significant impact on the codebase,
 - is inherently controversial; or
-- has failed to reach consensus amongst the Collaborators who are
+- has failed to reach a consensus amongst the Collaborators who are
   actively participating in the discussion.
 
-The Website group should serve as the final arbiter where required.
-
-## Code editing
+The Website group should be the final arbiter where needed.
+
+## Technologies used in the Website
+
+The Node.js Website is built upon [React][] and [Next.js][] respectively, the UI Rendering Engine and the Framework that builds the Website;
+
+The Website also uses several other Open Source libraries (not limited to) listed below:
+
+- Styling is done with [SCSS][] and CSS Modules
+- [TailwindCSS](https://tailwindcss.com/) is used as our CSS Framework and the Foundation of our Design System
+- [Hero Icons](https://heroicons.com/) is an SVG Icon Library used within our Codebase
+- [Shiki](https://github.com/shikijs/shiki) is a Syntax Highlighter used for our Codeboxes
+- [MDX][] and Markdown are used for structuring the Content of the Website
+- [`react-intl`][] is the i18n Library adopted within the Website
+- [`next-sitemap`](https://www.npmjs.com/package/next-sitemap) is used for Sitemap and `robots.txt` Generation
+- We use [Rehype](https://github.com/rehypejs/rehype) and [Remark](https://github.com/remarkjs/remark) to extend MDX functionality
```

Comment:
nit: any reason to switch between styles of markdown links here?

Reply:
We're using footnotes for any link used more than once.